composer.json

```diff
@@ -5,18 +5,22 @@
   "type"       : "wordpress-plugin",
   "license"    : "GPL-3.0-or-later",
   "require"    : {
-    "composer/installers": "~1.0",
-    "php": ">=7.1"
+    "composer/installers": "^1.12.0",
+    "php": "^8.0"
```

Comment:
Odd: in the Github Action matrix, you are testing for PHP 7.4 too, but here you are requiring 8.0+. 🤔

Reply:
Yeah, this should be updated to >= 7.4

Reply:
Fixed.